### ISSUE TITLE
Tags.Rating rationalisation

### ISSUE THREAD

**Issue by MikeZeDev** [manga-download/haruneko#298]
When writing plugins i asked myself : is there a need to have ```Tags.Rating.Erotica``` and ```Tags.Rating.Pornographic``` as far as a website is concerned? 

I think the idea is having a way to distinguish website mainly hosting +18 content, therefore a ```Tags.Rating.NSFW``` would cover both categories. Sometime the frontier between the category is kinda thin :D

I think ```Tags.Rating.Erotica``` and ```Tags.Rating.Pornographic``` exists for mangas and are fine, like in Mangadex classification

**Comment by ronny1982** [comment]
Aren't all manga website NSFW, i wouldn't like if my employees are having manga websites open while working ;)

The rating tags can technically be used for any media such as mangas and chapters as well, so the granulation could be fine.
That said, I don't see any reason against adding an additional NSFW tag.

**Comment by MikeZeDev** [comment]
> Aren't all manga website NSFW, i wouldn't like if my employees are having manga websites open while working ;)
I am in the same boat.

But i wouldnt really add a NSFW tag for Mangadex like i would for, idk, Nhentai, or those madara websites with only webtoons with well endowed females (to say the least).

>The rating tags can technically be used for any media such as mangas and chapters as well, so the granulation could be fine.

Granulation ok for medias, indeed. For websites there is no need for both "Erotica" & "Pornographic". I see if i find the time to change the tags.

**Comment by ronny1982** [comment]
Note: The content rating tags were copy & pasted from the [MangaDex API](https://api.mangadex.org/docs/3-enumerations/#manga-content-rating)

**Comment by MikeZeDev** [comment]
A warning could be handful : 

"Every manga website should be considered NSFW. The NSFW tag is a subjective tag and may not represent the actual rating of the website's content."

**Comment by 27136773** [comment]
The intention of `Tags` is it to make it easier for users to filter and categorize websites based on their major (intended or focused) content, not to provide legally accurate information about all its content.
The `Tags` are more like guidelines.

HakuNeko itself is NSWF, it doesn't matter which Maga/Hentai you read at work, you should never read at work, it will get you in trouble (except this is your job).

**Comment by MikeZeDev** [comment]
Of course. But perhaps we will be requested a "HideNSFW website" toggle so its just a shower thought.
